src/style.css

```diff
@@ -1,39 +1,45 @@
-body,
-h1,
-h2,
-ul,
-li {
+* {
   margin: 0;
   padding: 0;
+  box-sizing: border-box;
 }
 
+body {
+  /* background: url(./assets/bg.jpg);
+  background-repeat: no-repeat;
+  background-size: 100% 100%; */
```

Comment:
Please remove the commented code from the codebase for best practices. Commented code makes the codebase difficult to navigate and incomprehensible.